README.md

```
@@ -225,12 +232,17 @@ Set the git directory for future commands:
 gwt --repo /path/to/another/repo.git
 ```
 
-Remove a worktree and optionally its branch:
+Remove a worktree and clean up branches:
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)